index.html

```diff
@@ -75,23 +75,32 @@
 	{% assign posts = site.posts | sort:"name" %}
 	{% for post in posts %}
 		{% if post.role == 'talk' %}
-		<div class="talkers-one">
-			<a class="" href="{{ site.baseurl }}{{ post.url }}">
-				<div class="talk-info">
-						<h2 class="talker-name">
+		<div class="talkers-one" class="row">
+				<a class="col-xs-12 col-sm-6 col-md-4" href="{{ site.baseurl }}{{ post.url }}">
+					<div class="talk-info">
+            <p class="talk-image">
+							<img src="{{ site.baseurl }}/assets/images/speakers/{{ post.avatar }}"></img>
```

Comment:
`img` tags are self-closing, so: `<img src=".." class=".." />`. There is no `</img>`